facility-locator/map.js

```diff
@@ -13,7 +13,9 @@ FacilityLocator = (function() {
       directionsStart,
       geolocated = false; // Used when determining starting location for driving directions
       zoomLevelToShowMarkers = 5;
-      infowindows = {};
+      infowindows = {},
```

Comment:
The inconsistencies between commas and semicolons above really scares me, but that's not the fault of this PR.